test/integration/acceptance/fake_peer_example_test.cpp

```diff
@@ -274,14 +282,17 @@ TEST_F(FakePeerExampleFixture,
  * Check that after receiving a valid command the ITF peer sends either a
  * proposal or a batch to another peer
  *
+ * \attention the test is disabled because the ordering service protocol
+ * it uses was replaced by the new on-demand ordering version.
+ *
  * \attention this code is nothing more but an example of Fake Peer usage
  *
  * @given a network of two iroha peers
  * @when a valid command is sent to one
  * @then it must propagate either a proposal or a batch
  */
 TEST_F(FakePeerExampleFixture,
-       OrderingMessagePropagationAfterValidCommandReceived) {
+       DISABLED_OrderingMessagePropagationAfterValidCommandReceived) {
```

Comment:
Please create a task with explanation why the test is disabled and add a todo here.

Reply:
I found no task about removing the old ordering service code, so we decided the comment above the test code is enough.